Dockerfile

```diff
@@ -13,7 +13,6 @@ RUN go mod download
 
```

Comment:
Remove the 'go mod download'.  Instead, copy in the vendor/ dir.

Reply:
See https://github.com/DataWorkflowServices/dws-test-driver/commit/5b5b27ac01557286744ef5cd817d17ae9ea60a1d in [dean-from-pr1-a69a](https://github.com/DataWorkflowServices/dws-test-driver/tree/dean-from-pr1-a69a) where I've also added a make target 'container-unit-tests'.